src/components/Template/Template.tsx

```diff
@@ -152,10 +158,18 @@ const TemplateContentSidebar = (
 /**
  * This optional component should be the last child of the `Template`
  * component. This is rendered as an HTML `<footer>` element and spans the full
- * width of the page.
+ * width of the page. If an HTML `<footer>` element is already passed in a
+ * custom component, set `renderFooterElement` to `false`.
  */
-const TemplateFooter = (props: React.PropsWithChildren<TemplateProps>) => {
-  return <Box as="footer">{props.children}</Box>;
+const TemplateFooter = ({
+  children,
+  renderFooterElement = true,
+}: React.PropsWithChildren<TemplateFooterProps>) => {
+  return renderFooterElement ? (
+    <Box as="footer">{children}</Box>
```

Comment:
Can you add a warning message if a `footer` element is passed in as a child in this scenario?